examples/lband_decode/lband_decode.cc

```diff
@@ -0,0 +1,68 @@
+/**************************************************************************/ /**
+* @brief Example of decoding FusionEngine messages from a recorded file.
+* @file
+******************************************************************************/
+
+#include <cstdint>
+#include <cstdio>
+#include <fstream>
+
+#include <point_one/fusion_engine/messages/gnss_corrections.h>
+#include <point_one/fusion_engine/parsers/fusion_engine_framer.h>
+
+using namespace point_one::fusion_engine::messages;
+using namespace point_one::fusion_engine::parsers;
+
+static std::ofstream out_stream;
+
+// This is the callback for handling decoded FusionEngine messages.
+/******************************************************************************/
+void OnFEMessage(const MessageHeader& header, const void* data) {
+  // If the FusionEngine message is L-band data, process its payload.
+  if (header.message_type == LBandFrameMessage::MESSAGE_TYPE) {
+    auto frame = reinterpret_cast<const LBandFrameMessage*>(data);
+    auto lband_data =
+        static_cast<const char*>(data) + sizeof(LBandFrameMessage);
+    printf("Decoded %u L-band bytes.\n", frame->user_data_size_bytes);
+    out_stream.write(lband_data, frame->user_data_size_bytes);
+  }
+}
+
+/******************************************************************************/
+int main(int argc, const char* argv[]) {
+  if (argc != 2) {
+    printf("Usage: %s FILE\n", argv[0]);
+    printf("Decode L-band corrections and write contents to 'lband.bin'.");
+    return 0;
+  }
+
+  char buffer[2048];
```

Comment:
Minor suggestion: declare this down above the while loop where it's used so there's no confusion between this buffer size and the framing buffer size, which are unrelated.